custom/icds/views/custom_sms_report.py

```diff
@@ -18,26 +19,43 @@ class SMSUsageReport(BaseMessagingSectionView):
 
     @use_daterangepicker
     def dispatch(self, *args, **kwargs):
+        messages.info(self.request, _('Please note that this report takes a few hours to process.'))
```

Comment:
nit: might be worth a try to put this in the form itself. Not sure which one would look better.

Reply:
You mean to say like having a div with this message if the form?

Reply:
You can just add help_text, crispy form will render it on its own.